Px.Utils.UnitTests/PxFileTests/PxFileMetadataReaderTests/GetEncodingAsyncTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Px.Utils.UnitTests.PxFileTests.Fixtures;
+using PxUtils.PxFile.Meta;
```

Comment:
This namespace has a different naming convention compared to the one above it. PxUtils and Px.Utils.